apps/api/src/app/environments/environment.ts

```diff
@@ -89,6 +89,9 @@ const prodConfig = () => ({
     username: process.env.SYSLUMENN_USERNAME,
     password: process.env.SYSLUMENN_PASSWORD,
   },
+  airDiscountScheme: {
+    backendUrl: process.env.BACKEND_URL ?? 'http://localhost:4248',
+  },
```

Comment:
TODO: Get BACKEND_URL (and perhaps change name to ADS_BACKEND_URL) with the help of devops